test/WebSites/BasicWebSite/Views/Home/ViewWithPrefixedAttributeValue.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@{
+    Layout = "/Views/Shared/_Layout.cshtml";
```

Comment:
In this test, no reason to have a layout or `ViewBag.Title`, should be very precise.